tests/test_sklearn_decomposition.py

```diff
@@ -159,7 +159,7 @@ def test_pca_converter_two(self):
 
     # PLS regressor n_componenets two no scale
     def test_pca_converter_two_no_scale(self):
-        self._fit_model_pls_regressor(PLSR(n_components=10, scale=False))
+        self._fit_model_pls_regressor(PLSR(n_components=3, scale=False))
```

Comment:
Does this change anything?

Reply:
Yes, with the skl==1.1.1, you'll get [ValueError: n_components == 10, must be <= 3.](https://github.com/microsoft/hummingbird/runs/6892676674?check_suite_focus=true)